frontend/src/container/NewWidget/utils.ts

```diff
@@ -189,50 +189,6 @@ export const panelTypeDataSourceFormValuesMap: Record<
 			},
 		},
 	},
-	[PANEL_TYPES.HISTOGRAM]: {
```

Comment:
Interesting. How was this not flagged by ESLint locally or in the PR checks?

Reply:
Have to check the rules on this.